cirq/google/engine/engine.py

```diff
@@ -27,20 +27,15 @@
 import enum
 import random
 import string
-import sys
-import time
-from typing import Any, Callable, Dict, List, Optional, Sequence, TypeVar, \
+from typing import Dict, List, Optional, Sequence, TypeVar, \
     Union, TYPE_CHECKING
```

Comment:
Nit: should this line (or part of it) bump to the previous one now that it's shorter?